.github/workflows/maven.yml

```
@@ -15,28 +15,28 @@ on:
     branches: [ "master" ]
 
 jobs:
-  build-vaadin23:
+  build-vaadin24:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
     - name: Set up JDK
       uses: actions/setup-java@v3
```

actionlint (1.7.9)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)